sandy/fy.py

```diff
@@ -524,6 +633,41 @@ def to_hdf5(self, file, library):
 #        pass
 
 
+def _gls_setup(decay_data, kind, index_columns):
+    """
+    A function to obtain the prior sensitivity for performing GLS. 
+
+    Parameters
+    ----------
+    decay_data : `DecayData`
+        Container of radioactive nuclide data for several isotopes.
+    kind : `str`
+        Keyword for obtaining prior sensitivity.
+    index_columns : `str`
+        Keyword to determine if the sensitivity is the transposed one.
+
+    Raises
+    ------
+    TypeError
+        DESCRIPTION.
+
+    Returns
+    -------
+    S : `pandas.DataFrame`
+        The prior sensitivity for performing GLS.
+
+    """
+    if kind == 'cumulative':
+        S = decay_data.get_qmatrix()
+    elif kind == 'chain':
+        S = decay_data.get_decay_chains()
```

Comment:
this is not the sensitivity matrix for chain fission yields.

Reply:
The sensitivity matrix for chain fission yields are you meaning the matrix obtained with the method get_transition_matrix()?